pages/linux/btrfs.md

```diff
@@ -0,0 +1,23 @@
+# btrfs
+
+> A modern copy on write (CoW) filesystem for Linux.
+
+- Create subvolume:
+
+`sudo btrfs subvolume create {{path/to/subvolume}}`
+
+- List subvolume:
+
+`sudo btrfs subvolume list {{path/to/moount_point}}`
```

Comment:
Did you mean...
```suggestion
`sudo btrfs subvolume list {{path/to/mount_point}}`
```
Or moount_point is really something?

Reply:
oh yes, my mistake